src/exchange/binance-processor/commands/start.go

```diff
@@ -28,13 +28,11 @@ func startCommand() *cobra.Command {
 
 			sugarLogger := logger.Sugar()
 
-			viper, err := configuration.SetupConfigReader(".")
-			if err != nil {
+			var config configuration.Config
+			if err := entconfiguration.LoadConfig("config.yaml", &config); err != nil {
```

Comment:
Can we make this a bit more generic, I think it'd be good to be able to load specific config files with a prefix e.g. custody-config.yaml.
Thoughts?